File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.py
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        sumn= 0
        prodn= 1
        while n>0:
            sumn= sumn+(n%10)
            prodn= prodn*(n%10)
            n= n//10
        return (prodn-sumn)

File: 1281-subtract-the-product-and-sum-of-digits-of-an-integer/1281-subtract-the-product-and-sum-of-digits-of-an-integer.cpp
class Solution {
public:
    int subtractProductAndSum(int n) {
        int sumn= 0;
        int prodn= 1;
        while (n>0){
            sumn= sumn+(n%10);
            prodn= prodn*(n%10);
            n= static_cast<int>(n/10);
        }
        return (prodn-sumn);
    }
};